cli/lib/appdata.ts

```diff
@@ -11,15 +11,15 @@ export const snapshotSchema = z.object( {
 	atomicSiteId: z.number(),
 	localSiteId: z.string(),
 	date: z.number(),
-	name: z.string().optional(),
+	name: z.string(),
 	userId: z.number().optional(),
 } );
 
 const siteSchema = z
 	.object( {
 		id: z.string(),
 		path: z.string(),
-		name: z.string().optional(),
+		name: z.string(),
```

Comment:
I don't believe `name` props are actually optional for sites or snapshots. I made this change to accommodate the `reportKeyValuePair` logging of the snapshot name.